components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -10945,6 +10945,9 @@ components:
             Is Cross Tenant Subscriptions Enabled
           example: false
           default: false
+        authorizationHeader:
+          type: string
+          description: Authorization Header
```

Comment:
shall we add en example as well? we need to update the same in docs as well?

Reply:
Fixed.